src/libs/ReportUtils.js

```diff
@@ -317,7 +317,7 @@ function hasExpensifyEmails(emails) {
  * @return {Boolean}
  */
 function canShowReportRecipientLocalTime(personalDetails, report) {
-    const reportParticipants = lodashGet(report, 'participants', []);
+    const reportParticipants = _.without(lodashGet(report, 'participants', []), sessionEmail);
```

Comment:
So we need to also remove expensify emails from reportParticipants here

And also update the return check below

Reply:
@thienlnam we don't have to remove them as we already have a check for that 
https://github.com/Expensify/App/blob/f19a4d94870fdc96ea032d90217a03f9f5b46674/src/libs/ReportUtils.js#L324

Reply:
I'm mentioned it in the case that you have a group chat with Concierge and another User.  In this case, the timezone wouldn't show but you're just in a chat with another person and can have it show.  Does that make sense?

Reply:
Understood. So we will have to make a function which filters out expensify emails:
```js
function removeExpensifyEmails(emails) {
    return _.filter(emails, email => CONST.EXPENSIFY_EMAILS.indexOf(email) === -1);
}
```
We will have to use this function at both the places like we are doing for our own email

Reply:
@Puneet-here You can use `_.difference` instead of filter and then `indexOf`

Reply:
Only using _.difference will do the work.